Makefile

```diff
@@ -167,3 +167,28 @@ release:
 	@make release-kubectl-karmada GOOS=linux GOARCH=arm64
 	@make release-kubectl-karmada GOOS=darwin GOARCH=amd64
 	@make release-kubectl-karmada GOOS=darwin GOARCH=arm64
+
+.PHONY: install-addlicense
+## install-addlicense: check license if not exist install addlicense tools
+install-addlicense:
+ifeq (, $(shell which addlicense))
+	@{ \
+	set -e ;\
+	LICENSE_TMP_DIR=$$(mktemp -d) ;\
+	cd $$LICENSE_TMP_DIR ;\
+	go mod init tmp ;\
+	go get github.com/google/addlicense ;\
+	go install github.com/google/addlicense@latest ;\
+	rm -rf $$LICENSE_TMP_DIR ;\
+	}
+ADDLICENSE_BIN=$(GOBIN)/addlicense
+else
+ADDLICENSE_BIN=$(shell which addlicense)
+endif
+
+.PHONY: addlicense
+## addlicense: add license
+addlicense:
+	$(ADDLICENSE_BIN) -f hack/boilerplate/LICENSE_TEMPLATE -v \
```

Comment:
How did you come across this method? Just curious.

Reply:
I've used this way in previous projects haha 😸